administrator/components/com_joomlaupdate/views/default/view.html.php

```diff
@@ -102,7 +102,7 @@ public function display($tpl = null)
 		$this->phpSettings            = $model->getPhpSettings();
 		$this->nonCoreExtensions      = $model->getNonCoreExtensions();
 		$this->nonCoreCriticalPlugins = $model->getNonCorePlugins(array('system','user','authentication','actionlog','twofactorauth'));
-		$this->isBackendTeamplateIsis = (bool) $this->getModel()->isTemplateActive('isis');
+		$this->isBackendTemplateIsis = (bool) $this->getModel()->isTemplateActive('isis');
```

Comment:
```suggestion
		$this->isBackendTemplateIsis  = (bool) $this->getModel()->isTemplateActive('isis');
```
Align the "=" with those above

Reply:
Ah, and why don't you just use `$model` instead of `$this->getModel()`?